### DISCUSSION TITLE
Can someone give an example for bungeecord plugin messages?

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1861] [Help] [answered]
Can someone create an example for these 2 little things? 
```
Player#sendMessage(<playercount on all servers in the velocity>);
System.out.println(<playercount on all servers in the velocity>);
```
You can use this article if you know how to do it in spigot: https://www.spigotmc.org/wiki/bukkit-bungee-plugin-messaging-channel/#playercount

**Comment by mudkipdev** [comment]
Can you make a Velocity plugin first of all?

**Reply by Tofaa2** [reply]
https://github.com/Polar-Network/Polaroid/blob/master/src/main/java/net/polar/server/Polaroid.java#L169

Here's an example of how to use the bungee PM channel to send a player to another server

**Reply by DevBlokJochem** [reply]
no, I cannot make a velocity plugin

**Reply by DevBlokJochem** [reply]
> https://github.com/Polar-Network/Polaroid/blob/master/src/main/java/net/polar/server/Polaroid.java#L169
> 
> Here's an example of how to use the bungee PM channel to send a player to another server

thanks

**Reply by muelleel** [reply]
> https://github.com/Polar-Network/Polaroid/blob/master/src/main/java/net/polar/server/Polaroid.java#L169
> 
> Here's an example of how to use the bungee PM channel to send a player to another server

sadly the link is dead. Is there another link?

**Reply by Tofaa2** [reply, marked as answer]
A bit of a necro but yes the projects closed now, heres the gist of it

```java
public void bungeeServer(Player player, String server) {
  ByteArrayDataOutput out = ByteStreams.newDataOutput(); // Requires Guava IIRC
  out.writeUTF("Connect");
out.writeUTF(server);

player.sendPluginMessage("BungeeCord", out.toByteArray());
}

```

**Reply by muelleel** [reply]
Thanks, that seems to be the usual way of doing it, also within spigot plugins etc.
For me this did not work because I did not have the `ByteArrayDataOutput` class and also the Namespace did not work for me. 
But I figured it out, my solution, if anybody has also problems with the 'standard way', is here: https://github.com/Minestom/Minestom/discussions/1414#discussioncomment-7356584

**Comment by DevBlokJochem** [comment]
and how about the System.out.println()? So without a player
